### ISSUE TITLE
Can't delete future values

### ISSUE THREAD

**Issue by robinsonjj** [influxdata/influxdb#7798]
I can't delete future values using either the "delete from where tag = ''" syntax or the "drop series where tag = ''" syntax.

I have also tried using a 'delete from ... where time > now())' but this doesn't seem to help either.

Thanks

**Comment by jwilder** [comment]
We need more information in order to diagnose what is going on.  Can you update the issue description with the instructions listed in our [issue template](https://github.com/influxdata/influxdb/blob/master/.github/ISSUE_TEMPLATE.md)?

**Comment by robinsonjj** [comment]
Sure, sorry that was lazy of me.

I'm running version 1.1.0-1 on windows 7 and I'm using python to interact with influxdb:

starting out (printing length and earliest date):
```
In[317]: len(client.query("select * from forecasts_vd where id = 'ecphe_1_2'", database='db_v0.4')['forecasts_vd'])

Out[316]: 5403
In[318]: client.query("select * from forecasts_vd where id = 'ecphe_1_2'", database='db_v0.4')['forecasts_vd'].index.min()

Out[317]: Timestamp('2017-01-05 01:00:00+0000', tz='UTC')
```

running delete will delete past values but not future ones:
```
In[319]: client.query("delete from forecasts_vd where id = 'ecphe_1_2'", database='db_v0.4')

Out[318]: ResultSet({})
In[320]: len(client.query("select * from forecasts_vd where id = 'ecphe_1_2'", database='db_v0.4')['forecasts_vd'])

Out[319]: 5319
In[321]: client.query("select * from forecasts_vd where id = 'ecphe_1_2'", database='db_v0.4')['forecasts_vd'].index.min()

Out[320]: Timestamp('2017-01-06 17:00:00+0000', tz='UTC')
```

and the same output again if I use drop series. Let me know if I can provide any more information

**Comment by jwilder** [comment]
You may need to explicitly set the upper time bound with something like:

` ... time < now() + 24h`

The default end time defaults to `now()` still for deletes.  For selects, it was changed in #7519.  Looks like we need to fix deletes as well.

**Comment by jwilder** [comment]
@jsternberg

**Comment by robinsonjj** [comment]
That works, thanks!

**Comment by jwilder** [comment]
Fixed via #8329
